CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ option(ENABLE_QT2 "Enable Qt2 (use the other experimental Qt interface)" OFF)
 option(ENABLE_LTO "Enables Link Time Optimization" OFF)
 option(ENABLE_GENERIC "Enables generic build that should run on any little-endian host" OFF)
 option(ENABLE_HEADLESS "Enables running Dolphin as a headless variant" OFF)
+option(ENABLE_MICROPHONE "Enables Wii microphone support" ON)
```

Comment:
PortAudio is used for GameCube microphones, not Wii microphones. Wii (USB) microphones can only be used through USB passthrough with libusb.